src/main/java/socialite/ui/HelpWindow.java

```diff
@@ -15,8 +15,11 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String USERGUIDE_URL = "https://ay2122s1-cs2103t-f11-4.github.io/tp/UserGuide.html";
+    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL
```

Comment:
Does clicking on the url open a browser? If not I could create a browser class for interacting with the browser, then we could both use it (handles and this window)